Flaky test: TestMemoryWorkerCrash.tableAccessAfterWorkerCrash

See: https://github.com/prestodb/presto/actions/runs/12014468669/job/33490330202?pr=24141

Error stack:

```
Error:  Tests run: 33, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 46.275 s <<< FAILURE! - in TestSuite
Error:  com.facebook.presto.plugin.memory.TestMemoryWorkerCrash.tableAccessAfterWorkerCrash  Time elapsed: 2.545 s  <<< FAILURE!
java.lang.AssertionError: Expected query to fail: SELECT * FROM test_nation ORDER BY nationkey
	at org.testng.Assert.fail(Assert.java:110)
	at com.facebook.presto.tests.QueryAssertions.assertQueryFails(QueryAssertions.java:346)
	at com.facebook.presto.tests.AbstractTestQueryFramework.assertQueryFails(AbstractTestQueryFramework.java:305)
	at com.facebook.presto.plugin.memory.TestMemoryWorkerCrash.tableAccessAfterWorkerCrash(TestMemoryWorkerCrash.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
```

--- a/presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemoryWorkerCrash.java
+++ b/presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemoryWorkerCrash.java
@@ -17,8 +17,11 @@ import com.facebook.presto.server.testing.TestingPrestoServer;
 import com.facebook.presto.testing.QueryRunner;
 import com.facebook.presto.tests.AbstractTestQueryFramework;
 import com.facebook.presto.tests.DistributedQueryRunner;
+import com.google.common.collect.ImmutableMap;
 import io.airlift.units.Duration;
 import org.testng.annotations.Test;
+
+import java.util.Map;
 
 import static com.facebook.airlift.testing.Assertions.assertLessThan;
 import static io.airlift.units.Duration.nanosSince;
@@ -33,7 +36,8 @@ public class TestMemoryWorkerCrash
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return MemoryQueryRunner.createQueryRunner();
+        Map<String, String> coordinatorProperties = ImmutableMap.of("scale-writers", "false", "redistribute-writes", "true");
+        return MemoryQueryRunner.createQueryRunner(coordinatorProperties, ImmutableMap.of());
     }
 
     @Test

